05week/checkers.js

```diff
@@ -57,15 +83,100 @@ function Board() {
 function Game() {
 
   this.board = new Board();
+  this.moveChecker = function(whichPiece, toWhere){
```

Comment:
I would expect that the the board's grid attribute would not be accessed directly from here.